NYTimes.com.js

```diff
@@ -142,11 +142,12 @@ function scrape(doc, url) {
 		}
 
 		var authors = author.split(" and ");
-		for each(var author in authors) {
+		for (var j=0; j<authors.length; j++) {
+			var author = authors[j];
 			// fix capitalization
 			var words = author.split(" ");
-			for (var i in words) {
-				words[i] = words[i][0].toUpperCase() + words[i].substr(1).toLowerCase();
+			for (var k in words) {
```

Comment:
I didn't touch any of the `for ... in` except for variable renaming. I thought that `for ... in` is not deprecated and compatible among all browsers. Am I wrong?